onegov/election_day/formats/election/wabstic_proporz.py

```diff
@@ -69,34 +69,99 @@ def line_is_relevant(line, number, district=None):
         return line.sortgeschaeft == number
 
 
-def get_entity_id(line, entities):
-    entity_id = int(line.bfsnrgemeinde or 0)
-    return 0 if entity_id in EXPATS else entity_id
-
-
-def get_candidate_id(line):
-    return str(int(line.knr))
+def get_number_of_mandates(line):
```

Comment:
I tend to follow GO's dictum these days. For setter functions use `set_`, for getter functions omit the `get_`. I think we should adopt that in all new code.

See https://golang.org/doc/effective_go.html#Getters

Reply:
Agree. In this case I left it since otherwise var names conflict with function names and the following routine gets harder to read.